bin/phpunit

```diff
@@ -14,5 +14,8 @@ if (false === getenv('SYMFONY_PHPUNIT_VERSION')) {
 if (false === getenv('SYMFONY_PHPUNIT_DIR')) {
     putenv('SYMFONY_PHPUNIT_DIR='.__DIR__.'/.phpunit');
 }
+if (false === getenv('SYMFONY_DEPRECATIONS_HELPER')) {
+    putenv('SYMFONY_DEPRECATIONS_HELPER=weak_vendors');
```

Comment:
Weak_vendor is only for libraries. Let's borrow from https://github.com/symfony/recipes/pull/442/files instead